server/src/server.ts

```diff
@@ -181,9 +183,11 @@ export default function startServer(): void {
     app.get('/api/v2/organizations', userIsAuthorized, (req, res) => getOrganizations(req, res));
     app.post('/api/v2/organizations', (req, res) => createOrganization(req, res));
     app.get('/api/v2/organizations/:id', userIsAuthorized, (req, res) => getOrganization(req, res));
+    app.get('/api/v2/access/:code', userIsAuthorized, (req, res) => getOrganizationByAccessCode(req, res));
```

Comment:
```suggestion
    app.get('/api/v2/access/:code', (req, res) => getOrganizationByAccessCode(req, res));
```
userIsAuthorized macht an der Stelle keinen Sinn, da sich der User ja grade erst registrieren will!